R/model_functions.R

```diff
@@ -125,17 +133,18 @@ fit_bayesian_model <- function(focal_vs_comp, prior_param = NULL, run_shuffle =
 #' # and growth data to compare to
 #' data(posterior_param_ex, ex_growth_df)
 #'
-#' predictions <- focal_vs_comp_ex %>%
-#'   predict_bayesian_model(posterior_param = posterior_param_ex) %>%
+#' predictions <- posterior_param_ex %>%
+#'   predict(focal_vs_comp = focal_vs_comp_ex) %>%
```

Comment:
now that I think about it `predict.lm()` has the model as the first argument and the `newdata` as the second as well. I did wonder about this, given the tidyverse's propensity to have the data frame argument always come first